public/src/components/leaderboard/Leaderboard.tsx

```diff
@@ -84,12 +84,13 @@ export class Leaderboard extends React.Component<LeaderboardProps, LeaderboardSt
         const state = this.state;
 
         const leaderboardEntries = state.leaderboardEntries.map((entry, index) => (
-            <tr key={index}>
+            <tr key={index} className={entry.getIsblocked()?"leaderboard-blocked": ""}>
```

Comment:
Check class=`disabled` for this.

Reply:
Popups are prohibited for disabled so I wrote css for that